### DISCUSSION TITLE
[@types/dom-chromium-ai] Max temperature key missing on AILanguageModelCapabilities

### DISCUSSION THREAD

**Discussion by MiguelsPizza** [DefinitelyTyped/DefinitelyTyped#71025] [Issues with a @types package]
[Max Temperature is defined on the docs](https://github.com/explainers-by-googlers/prompt-api#:~:text=//%20Always%20null%20if%20available%20%3D%3D%3D%20%22no%22%0A%20%20readonly%20attribute%20unsigned%20long%3F%20defaultTopK%3B%0A%20%20readonly%20attribute%20unsigned%20long%3F%20maxTopK%3B%0A%20%20readonly%20attribute%20float%3F%20defaultTemperature%3B%0A%20%20readonly%20attribute%20float%3F%20maxTemperature%3B)

But not defined in the [declaration file ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f7e381701d4f97a8e31efeed2efddb27a1f665af/types/dom-chromium-ai/index.d.ts#L150)

```typescript
interface AILanguageModelCapabilities {
    readonly available: AICapabilityAvailability;

    readonly defaultTopK: number | null;
    readonly maxTopK: number | null;
    readonly defaultTemperature: number | null;
    readonly maxTemperature: number | null; // this is missing

    supportsLanguage(languageTag: Intl.UnicodeBCP47LocaleIdentifier): AICapabilityAvailability;
}
```

[a comment by an automated account is not shown]

**Comment by tomayac** [comment]
It is defined, but not yet implemented (here's the [bug](https://issues.chromium.org/issues/367771112)). Once it's actually exposed, it makes sense to have types for it. Do you want to create a draft PR for it, and we press merge once it's implemented? (FYI: @christianliebel)

**Reply by MiguelsPizza** [reply]
sure! Thanks for the quick response

**Reply by MiguelsPizza** [reply]
[Draft is up here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71027
)
